salesforce-gui/src/test/java/org/fundacionjala/salesforce/cucumber/stepdefs/personalInformation/EditStep.java

```diff
@@ -0,0 +1,36 @@
+package org.fundacionjala.salesforce.cucumber.stepdefs.personalInformation;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.fundacionjala.salesforce.ui.skins.SkinManager;
+import org.fundacionjala.salesforce.ui.skins.abstractPage.personalInformation.AbstractEditPersonalInformation;
+
+import java.util.Map;
+
+import org.testng.Assert;
+
+public class EditStep {
```

Comment:
Plaese, could you change this method for PersonalInformationSteps? because only edit is too generic